source/_integrations/reolink.markdown

```diff
@@ -66,13 +66,19 @@ Depending on the supported features of the camera number entities are added for:
 - AI vehicle sensitivity
 - AI pet sensitivity
 - Auto quick reply time
+- Auto track limit left
+- Auto track limit right
+- Auto track disappear time
+- Auto track stop time
 
 "Floodlight turn on brightness" controls the brightness of the floodlight when it is turned on internally by the camera (see "Floodlight mode" select entity) or when using the "Floodlight" light entity.
 
 When the camera is not moved and no person/pet/vehicle is detected for the "Guard return time" in seconds, and the "Guard return" switch is ON, the camera will move back to the guard position.
 
 When a Reolink doorbell is pressed the quick reply message from the "Auto quick reply message" select entity will be played after "Auto quick reply time" seconds, unless the "Auto quick reply message" is set to off.
 
+If the "Auto tracking" switch entity is enabled and a object disapears from view/stops moving for the "Auto track disappear time"/"Auto track stop time" in seconds, the camera goes back to its original position.
```

Comment:
```suggestion
If the "Auto tracking" switch entity is enabled, and a object disappears from view OR stops moving for the "Auto track disappear time"/"Auto track stop time", the camera goes back to its original position.
```
I'd recommend some minor changes to make it flow better.  Also, how about putting "All times are in seconds" somewhere else, since it presumably applies to all time configurations?